src/lib/cucumberjs/cucumber.js

```diff
@@ -79,8 +79,7 @@ class Cucumber {
       if (!this.cucumberChild.stopping) {
         log.debug('[chimp][cucumber] Cucumber not in a stopping state');
 
-        const result = jsonResults;
-        if (this.options.jsonOutput && JSON.parse(jsonResults).length) {
+        if (this.options.jsonOutput && jsonResults && JSON.parse(jsonResults).length) {
```

Comment:
@tiagolr we verified the fix for it on current Chimp master using the repo you provided in issue description. Can you try again using current Chimp master?